infrastructure/server/oidc_clients_fam.tf

```diff
@@ -31,7 +31,7 @@ resource "aws_cognito_user_pool_client" "fam_console_oidc_client" {
   }
 
   user_pool_id     = aws_cognito_user_pool.fam_user_pool.id
-  write_attributes = concat(var.minimum_oidc_attribute_list, ["custom:idp_display_name", "email", "custom:idp_business_id", "custom:idp_business_name"])
+  write_attributes = concat(var.minimum_oidc_attribute_list, ["custom:idp_business_name"])
```

Comment:
Why are we keeping FAM different? Should have a comment explaining why.

Reply:
There is no additional attribute added; they are still the same attributes as before for FAM.
I added a comment at the beginning of the [this TF file](https://github.com/bcgov/nr-forests-access-management/pull/1792/files#diff-0234a9835dc96b38911131d11fc726158f9e043243821d6766060a0fd8c9ec72).

Reply:
My question is why does FAM require idp_business_name and will this work given what we know know about other apps not having this attribute listed means the user isn't guaranteed to have this stored in the FAM_USER table...

In other words, if our goal in the PR is to handle attributes consistently, are we clear on why FAM is different and confident that it will work.

Reply:
Sorry, just saw the comment on why FAM needs idp_business_name, so the question remaining is whether this will work...

Reply:
Okay, I've assessed. FAM uses idp_business_name only in the front-end, extracted from the identity token, and the FAM_USER table doesn't even have a column to store idp_business_name. Therefore, it is safe to have only FAM add this extra attribute, with the understanding it isn't stored in the FAM_USER table. All attributes of the user stored in FAM_USER table must be part of the minimum attribute set used by all applications.